src/components/ClipListItem/ClipListItem.tsx

```diff
@@ -119,23 +122,35 @@ export const ClipListItem = ({
   return (
     <>
       <li className='clip-list-item'>
+        {showMoveButton &&
+          <div className='side-wrapper'>
+            <ButtonCircle
+              ariaLabel={t('Move')}
+              className='move'
+              faIcon={faArrowsAltV}
+              iconOnly
+              size='medium'
+              {...dragHandleProps}
+            />
+          </div>
+        }
         <div className={isChapter ? 'main-wrapper chapter-main-wrapper' : 'main-wrapper'}>
           {isChapter && showImage && finalImageUrl && (
             <PVImage alt='' height={PV.Images.sizes.medium} src={finalImageUrl} width={PV.Images.sizes.medium} />
           )}
           {itemMainSection}
         </div>
-        {showRemoveButton && (
+        {(showRemoveButton) && (
           <div className='side-wrapper'>
-            <ButtonCircle
+            {showRemoveButton && <ButtonCircle
```

Comment:
If this is checked above (on line 143), does it need to be checked again here?

Reply:
You're right. I had the drag button on the right-side first, and therefore had those extra checks bout in the wrapping div, and around each button. But I guess I forgot to clean it up once I moved the drag button to the left side